### PR TITLE
multi-factor authentication

### PR DESCRIPTION
**Pull request by brianteeman**
we've agreed to use multi-factor as a hyphenated word and to use this term instead of two factor authentication, TFA, 2FA

### CODE REVIEW

File: A-Z.md [hunk 1]
```diff
@@ -124,3 +127,6 @@ Tooltips should:
 
 ### Trademarked Name
 This should always be written with a capital letter unless the trademark is for lowercase eg iPhone.
+
+### Two Factor Authentication
+This should not be used. See multi-factor authentication.
```
Comment (nikosdion) [inline A-Z.md:132]: Capitalization. Should be “Multi-factor Authentication”.

File: words2watch.md [hunk 2]
```diff
@@ -65,6 +65,7 @@ Words to Watch (W2W) are words commonly misspelled or which may have variations
 | meta [attribute]                            |meta-description                  |Always two words and [attribute] is always lowercase|
 | misspell                                    | mispell                          | Spell with a double s.                                                                                                                                                                                                                                                    |
 | Module                                      | module                           | Always capitalise when used for a Joomla! Extension.                                                                                                                                                                                                                      |
+| multi-factor authentication                 | multifactor authentication multi factor authentication | multi-factor must always be a single hyphenated word |
```
Comment (nikosdion) [inline words2watch.md:68]: Capitalization. Should be “Multi-factor Authentication”.

**Review by nikosdion — COMMENTED** [review]
Capitalisation concerns.

File: words2watch.md (hunk 2 above)
Comment (brianteeman) [reply words2watch.md:68]: good point - I will update accordingly - thanks

**Comment by wilsonge** [comment]
Thanks!

**Pull request merged**